Formula/mariadb@10.5.rb

```diff
@@ -36,7 +36,9 @@ class MariadbAT105 < Formula
   keg_only :versioned_formula
 
   # See: https://mariadb.com/kb/en/changes-improvements-in-mariadb-105/
-  deprecate! date: "2025-06-01", because: :unsupported
+  # and https://mariadb.org/about/#maintenance-policy
+  # End-of-life on 2025-06-24. Disable date set to 3 months after.
+  disable! date: "2025-09-24", because: :unsupported
```

Comment:
Use official EOL date from https://mariadb.org/about/#maintenance-policy +3 months for disable date to allow users to get final release while also scheduling removal given large number of `mariadb` versioned formulae.

Release series | Stable (GA) Date | End-of-life
-- | -- | --
10.5 | 24 Jun 2020 | **24 Jun 2025**

Reply:
`disable!` automatically outputs deprecated in `brew info mariadb@10.5`, so will probably revert to `deprecate!` instead.